torch/csrc/jit/codegen/cuda/graph_fuser.cpp

```diff
@@ -940,7 +940,11 @@ struct CudaGraphFuser {
       // extended shape expression support to reduction operations
       // TODO: `aten::sum` is too flexible, we should restrict for a better
       // match
-      if (n->kind() == aten::sum) {
+      // TODO: Add python tests where we check for existing ops and their
+      // shape expression logic.
+      static std::unordered_set<Symbol> reduction_ops(
+          {aten::sum, aten::mean, aten::var, aten::std});
```

Comment:
:+1:

Reply:
This looks about right. Since we've already have python tests for aten::sum, I don't think it's necessary to duplicate that for the rest of the ops.